### PR TITLE
Misc. changes to docs and code to follow recommended install

### PR DESCRIPTION
**Pull request by bcipolli**
I had some issues installing the server with the given documentation. I also took the liberty of reorganizing a bit. Here's what I did:

Needed:
- `pip install -r requirements.txt` must be done after mysql is installed.
- There's some `libssl` funkiness that I hit (and others are likely to hit)
- `python manage.py migrate` should be run after `settings_local.py` is updated with database info.
- The database python snippet had a syntax error.
- Added a link to `/admin` in the test homepage. The instructions suggest accessing the admin by going to the homepage, but there was no link from there to the admin panel. 
- Added `src` to `.gitignore`; it is conda's workspace directory (like virtualenv).
- When I ran the migration management command, Django mentioned there's a pending migration. So I ran the command to create it, and committed it here.

Optional:
- Moved the mock-ups into the wiki. Added text giving an overview of the project and linking to the technical description and mock-ups. I found the mock-ups to be less important to understanding this back-end app (even though they're extremely important in the overall project)
- Reorganized / reformatted the outline a bit.
- I removed `$` from commands; they were only in some commands, and it makes it more challenging to copy-paste (can't just triple-click on the text), and it's pretty clear it's to be run from the command-line already

Glad to iterate on this if something's not looking good. I tested this all manually, by running each command one-by-one (to get my server up & running with data!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,26 @@ Status](https://travis-ci.org/caciviclab/disclosure-backend.svg?branch=master)](
...
 California Civic Lab Disclosure Backend
 ==================================================
-[![Mockup](/mockups/odca-mobile-09-ballot_measure-small.jpeg)](/mockups/odca-mobile-09-ballot_measure.jpeg)
-[![Mockup](/mockups/odca-mobile-09-ballot_measure 2-small.jpeg)](/mockups/odca-mobile-09-ballot_measure 2.jpeg)
-[![Mockup](/mockups/odca-mobile-09-ballot_measure 3-small.jpeg)](/mockups/odca-mobile-09-ballot_measure 3.jpeg)
-[![Mockup](/mockups/odca-mobile-09-ballot_measure 4-small.jpeg)](/mockups/odca-mobile-09-ballot_measure 4.jpeg)
-[![Mockup](/mockups/odca-mobile-09-ballot_measure 5-small.jpeg)](/mockups/odca-mobile-09-ballot_measure 5.jpeg)
-[![Mockup](/mockups/odca-mobile-09-ballot_measure 6-small.jpeg)](/mockups/odca-mobile-09-ballot_measure 6.jpeg)
-[![Mockup](/mockups/odca-mobile-09-ballot_measure 7-small.jpeg)](/mockups/odca-mobile-09-ballot_measure 7.jpeg)
-[![Mockup](/mockups/odca-mobile-09-ballot_measure 8-small.jpeg)](/mockups/odca-mobile-09-ballot_measure 8.jpeg)
```
Comment (tdooner) [inline README.md:13]: I appreciate that you moved these to the Wiki instead of removing them outright, but I still think they are important to remain in the README. It's the first place people expect to find information about the project, and we don't quite have enough information to organize to fully utilize the wiki structure.

How would you feel about moving these a bit lower in the README instead of removing them entirely?

File: README.md [hunk 2]
```diff
@@ -40,33 +39,36 @@ First, clone `disclosure-backend` (or your fork of it) to your own local copy.
...
+  mysql> \q
+  ```
 
-```
-mysql -p --user root
-mysql> create database calaccess_raw;
-mysql> \q
-python manage.py migrate
-```
+2. Modify `settings.py` (or create `settings_local.py`)
```
Comment (tdooner) [inline README.md]: Just a tiny nit: I think we should reword this to only encourage `settings_local.py` because settings.py is checked in.

File: README.md [hunk 3]
```diff
@@ -78,56 +80,93 @@ DATABASES = {
...
+  ```
+  
+  OSX: If you get the following error `django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module: dlopen(_mysql.so, 2): Library not loaded: libssl.1.0.0.dylib`
+  
+  Then, you need to add openssl to your `DYLD_LIBRARY_PATH`:
+  1. Go to `/usr/local/Cellar/openssl/`, and locate your directory (e.g. 1.0.2d_1)
+  2. Add the following to your `~/.bash_profile`:
+   ```
+   export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:/usr/local/Cellar/openssl/1.0.2d_1/lib
+   ```
```
Comment (tdooner) [inline README.md:110]: Wow, that's messed up that you have to do that. Homebrew really doesn't take care of this thing for you?

File: README.md (hunk 1 above)
Comment (bcipolli) [inline README.md:13]: Sounds good to me. You mind if I leave the link to the wiki, at the top?

File: README.md (hunk 3 above)
Comment (tdooner) [inline README.md:110]: Also, I haven't had to do this on the couple Mac OS laptops that I've installed python projects on, hmmmm.

File: README.md (hunk 2 above)
Comment (bcipolli) [inline README.md]: I agree!

File: README.md [hunk 4]
```diff
@@ -78,56 +80,93 @@ DATABASES = {
...
-python manage.py runserver
-```
+## Deploying the app
 
-Then go to http://127.0.0.1:8000/admin to log in and see data.
+  For deployment to the official website:
 
+  ```
+  ssh opencal.opendisclosure.io /usr/local/bin/deploy-backend
+  ```
```
Comment (tdooner) [inline README.md]: Another nit: The two-space indentation of each section's contents seems unnecessary.

File: disclosure_backend/project/views.py [hunk 5]
```diff
@@ -67,7 +67,11 @@ def location_view(request, fips_id):
 
 
 def homepage_view(request):
-    return HttpResponse("<a href='/docs/'>Check out the API Documentation</a>")
+    return HttpResponse("""
+        <a href='/docs/'>Check out the API Documentation</a>
+        <br/>
+        <a href='/admin/'>View the admin interface / database data.</a>
+    """)
```
Comment (tdooner) [inline disclosure_backend/project/views.py:74]: Nice.

**Comment by tdooner** [comment]
Cool. I don't know django well-enough to know if that migration is the right thing to do, but it seems fine if it made things work for you. :+1:

File: README.md (hunk 3 above)
Comment (bcipolli) [inline README.md:110]: hmmm. were you running on El Capitan? Perhaps you had an earlier version of MySql running?

I'm OK to remove this. I got the sense from doing searches that a number of people are hitting this issue on El Capitan.

**Comment by bcipolli** [comment]
@tdooner I'm pretty confident about the migration; if the migrations are current, Django doesn't complain and running the migration command produces no outputs. So, I'm relatively confident it was needed.

I updated per your comments, then squashed the `README` commits to a single commit. LMK if you'd like to see any other changes.

Edit to add "relatively" confident; who knows, maybe Django changed a lot since 1.5 :smile_cat:

**Comment by tdooner** [comment]
Thanks dude, this looks great. It will definitely help the next people who want to get involved!

**Comment by bcipolli** [comment]
Thanks @tdooner !
